apps/frontend/src/app/(app)/space/layout.tsx

```diff
@@ -60,7 +57,7 @@ const SpaceLayout: React.FC<Props> = ({ children }) => {
   }
 
   const constructPath = (spaceName: string) => {
-    return `space/${spaceName.toLowerCase().replace(/\s+/g, '-')}`
+    return `space/${spaceName.toLowerCase().replace(/\s+/g, "-")}`
   }
```

Comment:
Try adding some comments on logics like this for better understanding about what the code does